src/Impostor.Server/Net/State/Game.Api.cs

```diff
@@ -97,5 +98,19 @@ public async ValueTask SetPrivacyAsync(bool isPublic)
                 await SendToAllAsync(writer);
             }
         }
+
+        public async ValueTask StartGameAsync()
```

Comment:
OUT OF SCOPE: It's kind of nasty how this code is duplicating code in Game.Incoming.cs